packages/wrangler/package.json

```diff
@@ -36,6 +36,7 @@
     "cli"
   ],
   "dependencies": {
+    "blake3-wasm": "^2.1.5",
```

Comment:
could you tell me why we use `blake3-wasm` here, and not `xxhash` which we already use for sites? If possible, I'd like to use xxhash everywhere, unless there's a reason I'm not aware of?

Reply:
We use Blake3 in Pages currently.

Reply:
I did come across a pure-JS implementation later on for something else, so we could swap that in, in place of the WASM, if you'd prefer.

Reply:
Oh I see, so that hashes match what Pages already does internally. But since the projects published via API wouldn't be compared with "native" Pages projects, would the choice of hasher here matter? Just want to make sure we're intentional here with this choice, because changing later would invalidate all previous projects (if I'm understanding correctly)

Reply:
There's a possible future where you can upload to an existing project, so yeah, we need to be consistent with how files are hashed today.

Reply:
Aight, that's a good reason.